tests/test_ibl_converter.py

```diff
@@ -0,0 +1,154 @@
+from pynwb import NWBHDF5IO, TimeSeries
+from .utils import *
```

Comment:
I'd rather not use `import *` outside of `__init__.py` files.

Reply:
solved